e2e/cases/shims/index.test.ts

```diff
@@ -1,4 +1,5 @@
 import { join } from 'node:path';
+import { describe } from 'node:test';
```

Comment:
why from node test

Reply:
😅 intellisense is not intelligent enough